src/components/Contacts/ContactPartnershipStatus/ContactLateStatusLabel/getDonationLateStatus.ts

```diff
@@ -0,0 +1,85 @@
+import { DateTime } from 'luxon';
+
+export enum ContactLateStatusEnum {
+  OnTime,
+  LateLessThirty,
+  LateMoreThirty,
+  LateMoreSixty,
+}
+
+const selectLaterDate = (lateAt: string, pledgeStartDate: string): string => {
+  return lateAt > pledgeStartDate ? lateAt : pledgeStartDate;
+};
+
+const getStatusFromDays = (daysDiff: number): ContactLateStatusEnum => {
+  if (daysDiff < 0) {
+    return ContactLateStatusEnum.OnTime;
+  }
+  if (daysDiff < 30) {
+    return ContactLateStatusEnum.LateLessThirty;
+  }
+  if (daysDiff < 60) {
```

Comment:
As the last enum us called `LateMoreSixty`, I think this if should read less or equal to 60, then `LateMoreThirty`.


```suggestion
  if (daysDiff <= 60) {
```